tools/functional-tester/etcd-tester/checks.go

```diff
@@ -87,29 +89,93 @@ func (lc *leaseChecker) Check() error {
 		return err
 	}
 	plog.Infof("checking short lived leases %v", lc.ls.shortLivedLeases.leases)
-	return lc.check(true, lc.ls.shortLivedLeases.leases)
+	return lc.shortLivedLeaseCheck()
+
+}
+
+// shortLivedLeaseCheck verifies the expiry of shortLivedLeases.
+// the logic is follwing:
+//		1. for each lease from shortLivedLeases map
+//			1. get lease remaining TTL info.
+//			2. if lease is not expired, then wait until it expires.
+//			3. if lease is expired but not deleted, then wait for 1 second for etcd cluster to process lease deletion.
+//			4. check lease expiry.
+//				1. retry again by repeating steps 1 - 4 if check fails
+//				2. exit if check succeeds
+//			5. return error if all retries have failed
+//		2. return any errors on checking lease from shortLivedLeases
+func (lc *leaseChecker) shortLivedLeaseCheck() error {
```

Comment:
`func (lc *leaseChecker) checkShortLivedLeases() error` to be consistent with the other check* functions.